### ISSUE TITLE
Multiplayer Transforming bug

### ISSUE THREAD

**Issue by PDabro** [MoriyaShiine/bewitchment#84]
Alright so, I'm werepyre from Besmirchment addon and my friend is a werewolf. When i play alone at the server everything works fine, but when my friend joins to the server - his werewolf model is not working for him, but I see him as a werewolf. When my friend is on the server and when im reconnecting back to the server I have the same issue, my werepyre model is bugged, but my friend can see me as werepyre. 
![2021-04-08_13 04 26](https://user-images.githubusercontent.com/12714045/114042632-e2ea2480-9885-11eb-8a3d-f898ed93edf7.png)
![2021-04-08_13 06 07](https://user-images.githubusercontent.com/12714045/114042637-e4b3e800-9885-11eb-8f9a-b22796f6b107.png)
![2021-04-08_13 06 12](https://user-images.githubusercontent.com/12714045/114042648-e67dab80-9885-11eb-8e1e-be910d0c455e.png)

Its bugged too without Besmirchment addon, so the issue is inside Bewitchment. I talked with Aelpecyem and he told me to report this weird bug here. 

in short, when im playing alone everything works fine as werewolf/werepyre, when theres more than one player, its bugged

For more info you can DM me on discord: GreyOkay#9189

Edit: yup I'm 100% sure if there's more than 1 guy who can transform to something its getting bugged
(its like server dont know youre a werepyre/werewolf/vampire/etc cuz when its bugged, even my patron attacks me and Im getting stuck in salt)

**Comment by MoriyaShiine** [comment]
Does this still happen and if so, can you provide a log

**Comment by PDabro** [comment]
I don't receive any logs on console when transforming

**Comment by MoriyaShiine** [comment]
I don't need it when transforming, I need the full log

**Comment by PDabro** [comment]
[latest.log](https://github.com/MoriyaShiine/bewitchment/files/6386088/latest.log)

you mean this?

**Comment by MoriyaShiine** [comment]
Yes, thank you

I have no idea how the game functions in offline mode (regarding storing data, since UUIDS are not stored) so please try using an actual copy of the game.

**Comment by Rover656** [comment]
This happens on my server also, however that is running in online mode. I don't have logs to hand at this moment however.

**Comment by ZroNoh** [comment]
This happens on my server that is running in online mode as well.
Here's my logs
https://mclo.gs/LlIvt5i
